### DISCUSSION TITLE
Can Legendary Launcher get you banned on Fortnite?

### DISCUSSION THREAD

**Discussion by ghost** [legendary-gl/legendary#316] [Q&A]
Can Legendary Launcher get you banned on Fortnite?

**Comment by derrod** [comment]
Probably not.

**Reply by ghost** [reply]
What is your proof? Bc i havve seen ppl getting banned

**Reply by derrod** [reply]
Where?

It's probably not because they used legendary, but because they're general dickbags (cheaters and the likes).

**Reply by JordanPlayz158** [reply]
Yeah, Anticheats deal with bans, all legendary does is pass the info to the game to authenticate so if the anticheat bans them then it'd be because they are cheating/using something to trip the anticheat
